src/common/html-attributes/index.js

```diff
@@ -12,21 +12,24 @@ function camelToKebab(s) {
  * Create object of HTML attributes for pass-through to the DOM
  * @param {Object} input
  */
-function processHtmlAttributes(input = {}) {
+function processHtmlAttributes(input = {}, ignore) {
```

Comment:
Since we're passing in the entire input now, there probably isn't a case where it is undefined. Maybe worth removing the default parameter to an empty object?